XenAdmin/TabPages/ManageUpdatesPage.Designer.cs

```diff
@@ -323,12 +335,32 @@ private void InitializeComponent()
             // tableLayouPanel5
             // 
             resources.ApplyResources(this.tableLayouPanel5, "tableLayouPanel5");
+            this.tableLayouPanel5.Controls.Add(this.dataGridViewHosts, 0, 1);

```

Comment:
A TableLayoutPanel can have one item per cell. In cases like this with two GridViews in the same cell, one normally adds a plain Panel in the cell with Fill and all margins and paddings set to 0, and puts the two GridViews in the Panel with Fill. To work on either on those on the designer you can bring them to the foreground by selecting them on the document outline.